examples/controls_example/public/app/react_control_example.tsx

```diff
@@ -207,18 +217,15 @@ export const ReactControlExample = ({
     };
   }, [controlGroupApi, timeslice$]);
 
-  if (error || (!dataViews?.[0]?.id && !loading))
-    return (
-      <EuiEmptyPrompt
-        iconType="error"
-        color="danger"
-        title={<h2>There was an error!</h2>}
-        body={<p>{error ? error.message : 'Please add at least one data view.'}</p>}
-      />
-    );
-
-  return loading ? (
+  return loadingDataView ? (
```

Comment:
I don't think you need to block rendering of the control group until data view is loaded. Each control is in charge of loading their own data views.